src/background/index.js

```diff
@@ -160,4 +160,13 @@ chrome.commands.onCommand.addListener((command) => {
       });
       break;
   }
+});
+
+chrome.browserAction.onClicked.addListener(function (tab) {
+  chrome.tabs.query({ active: true, currentWindow: true }, (results) => {
```

Comment:
This query to get the current tab is not necessary as the above callback for `browserAction.onClicked` passes `tab` object which can be directly used.

Reply:
True. Taking care of this in the next commit.